src/components/dataset/query/QueryView.jsx

```diff
@@ -99,6 +107,15 @@ export class QueryView extends React.PureComponent {
           PAGE_SIZE,
         ),
       );
+      const bq = new BigQuery();
+      bq.getTotalRows(
+        dataset.dataProject,
+        `#standardSQL
+          SELECT COUNT(*) FROM \`${dataset.dataProject}.datarepo_${dataset.name}.${selected}\` AS ${selected}
```

Comment:
I would suggest `COUNT(1)` instead of `COUNT(*)`. `COUNT(*)` will scan every column in the table. `COUNT(1)` won't scan anything for counting, so you're only paying for the columns mentioned in the joins and filters.